src/test/java/gameboy/gamepack/racinggame/data/entity/CarTest.java

```diff
@@ -0,0 +1,20 @@
+package gameboy.gamepack.racinggame.data.entity;
+
+import org.junit.jupiter.api.*;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+
+class CarTest {
+
+    @Test
+    @DisplayName("초기 Car의 run 결과 테스트")
```

Comment:
```suggestion
    @DisplayName("자동차 이동 테스트")
```
테스트 설명이 너무 애매한 것 같아요 정확하게 표현하면 좋을 것 같습니다.